python/ecl/util/util/ctime.py

```diff
@@ -50,7 +50,7 @@ def date(self):
         return datetime.date(*self.time()[0:3])
 
     def datetime(self):
-        return datetime.datetime(*self.time()[0:6])
+        return datetime.datetime.utcfromtimestamp(self.ctime())
```

Comment:
Maybe not measurably faster, but by inlining `self.ctime()` to `self.value()` you should save a function call.